daemon/api.go

```diff
@@ -66,6 +66,12 @@ var api = []*Command{
 	assertsFindManyCmd,
 }
 
+// PolicyKit action names
+const sideloadSnapAction = "com.ubuntu.snappy.sideload-snap"
```

Comment:
I just copied the associated method name sideloadSnap() - if there's a more appropriate name then happy to switch to that.